erica/infrastructure/sqlalchemy/alembic/env.py

```diff
@@ -0,0 +1,83 @@
+import os
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+from erica.infrastructure.sqlalchemy.erica_request_schema import EricaRequestSchema
+
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
```

Comment:
Just saw that now: We might want to use the same logging configuration as for the erica_app. You could have a look at the steuerlotse repo, where we use the logging.json (https://github.com/digitalservice4germany/steuerlotse/blob/main/webapp/migrations/env.py).

Reply:
👍🏻  I will have a look.

Reply:
new ticket to integrate overall logging?

Reply:
[Ticket](https://steuerlotse.atlassian.net/browse/STL-2074)